app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Cwiczenie</title>
+  <%= stylesheet_link_tag    'admin', media: 'all' %>
+  <%= javascript_include_tag 'admin' %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+    <%= render partial: 'admin/navbar' %>
+    <%= render partial: 'shared/flash' if !flash.empty?  %>
```

Comment:
It is necessary ? I think it shouldn't display anything when flash is empty as well, so this condition can be removed.